Av1an/chunk_queue.py

```diff
@@ -147,7 +147,10 @@ def create_video_queue_select(args: Args, split_locations: List[int]) -> List[Ch
     """
     # add first frame and last frame
     last_frame = frame_probe(args.input)
-    split_locs_fl = [0] + split_locations + [last_frame]
+    if 0 not in split_locations:
+        split_locs_fl = [0] + split_locations
+    if last_frame not in split_locations:
+        split_locations = split_locations + [last_frame]
```

Comment:
Why is this needed? `split_location` will never have the first or last frame included.

Reply:
I am not sure about the first frame, but the last frame gets added by extra_split when using --extra_split option, at least for some of my videos. I added it as a quick fix and it is my first time working with github and pull requests, it was not supposed to be included in this pull request, and I noticed it too late. Sorry for that. We should probably look at extra_split for this issue.

Reply:
For sure extra splits should not exists in a way it is currently

Reply:
Ok. We should take a look at extra splits. The reason I asked about this is that this is not the only place that the first and last frame is added to the split locations, so the other place might need changing as well.